ethereum-consensus/src/deneb/blinded_blob_sidecar.rs

```diff
@@ -10,7 +10,7 @@ pub struct BlindedBlobSidecar<const BYTES_PER_BLOB: usize> {
     pub block_root: Root,
     #[serde(with = "crate::serde::as_string")]
     pub index: BlobIndex,
```

Comment:
the dev ux here isn't great but we also need to tell `serde` to write this as a string
```suggestion
    pub index: BlobIndex,
    #[serde(with = "crate::serde::as_string")]
```

Reply:
ah yeah, updated! Thanks